ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -478,6 +478,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb
+- as: aws-ipi-kubevirt-hypershift-p3-f21
```

Comment:
@LiangquanLi930 following previous name rule, it should be `aws-ipi-kubevirt-hypershift-mgmt-p3-f21`, like `aws-ipi-ovn-hypershift-mgmt-p3-f21`